src/viewer/App.tsx

```diff
@@ -412,6 +415,24 @@ export const App = observer(({store}: { store: AppStateStore }) => {
   }
   const {frames, activeFrame} = connection;
   const active = frames.find(f => f.id === activeFrame);
+
+  const Row = ({ index, style, data }: ListChildComponentProps) => {
+    const frame = data[index];
```

Comment:
@SerCeMan 
Sorry I just realized, that I broke that line.

It should access `frames[index]` not `data[index]` (the later would only work if [`itemData` prop would be set on `<FixedSizeList />`](https://react-window.vercel.app/#/api/FixedSizeList))